src/site/includes/metatags.drupal.liquid

```diff
@@ -0,0 +1,71 @@
+<link rel="canonical" href="{{ hostUrl }}{{ entityUrl.path }}" />
+{% if changed %}
+<meta name="DC.Date" content="{{changed| dateFromUnix: 'YYYY-MM-DD'}}" property="http://purl.org/dc/terms/date">
+{% endif %}
+{% assign tagsSize = entityMetatags | size %}
+{% if tagsSize > 0 %}
+  <!-- Metatags -->
+  {% for tag in entityMetatags %}
+    {% if tag.__typename == "MetaValue" %}
+<meta name="{{ tag.key }}" content="{{ tag.value }}">
+    {% else %}
+      {% if tag.__typename == "MetaProperty" %}
+<meta property="{{ tag.key }}" content="{{ tag.value }}">
+      {% else %}
+        {% if tag.__typename == "MetaLink" %}
+<link rel="{{ tag.key }}" href="{{ tag.value }}">
+        {% endif %}
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+{% else %}
+<meta property="og:site_name" content="VA.gov">
+<meta name="twitter:card" content="Summary">
+<meta name="twitter:site" content="@DeptVetAffairs">
+<meta name="twitter:image" content="{{ hostUrl }}/img/design/logo/va-og-twitter-image.png">
+  {% if title %}
+<meta content="{{ title }} - VA.gov" property="og:title">
+<meta content="{{ title }} - VA.gov" name="twitter:title" >
+  {% else %}
+<meta content="{{ hostUrl }}" property="og:title">
+<meta content="{{ hostUrl }}" name="twitter:title">
+  {% endif %}
+<meta content="website" property="og:type">
+  {% if entityUrl.path %}
+<meta content="{{ hostUrl }}/{{ entityUrl.path }}" property="og:url">
+  {% endif %}
+  {% if fieldDescription %}
+    {% assign description = fieldDescription | newline_to_br %}
+  {% else %}
+    {% if fieldIntroText %}
+      {% assign description = fieldIntroText.processed | strip_html %}
+    {% else %}
+      {% if fieldIntroTextNewsStories %}
+        {% assign description = fieldIntroTextNewsStories.processed  | strip_html %}
+      {% else %}
+        {% if fieldIntroTextEventsPage %}
+          {% assign description = fieldIntroTextEventsPage.processed | strip_html %}
+        {% else %}
+          {% if fieldClinicalHealthServi %}
+            {% assign description = fieldClinicalHealthServi.processed | strip_html %}
+          {% else %}
+            {% if fieldLocationsIntroBlurb %}
+              {% assign description = fieldLocationsIntroBlurb.processed | strip_html %}
+            {% else %}
+              {% if fieldPatientFamilyServicesIn %}
+                {% assign description = fieldPatientFamilyServicesIn.processed | strip_html %}
+              {% endif %}
+            {% endif %}
+          {% endif %}
+        {% endif %}
+      {% endif %}
+    {% endif %}
+  {% endif %}
```

Comment:
I think this may be easier to read using [`elsif`](https://help.shopify.com/en/themes/liquid/tags/control-flow-tags).

Reply:
Thanks. I tried 'elseif' and that didn't work, so I assumed liquid didn't have that capability. Should have RTFM.
I'll update.

Reply:
Haha I had to look up `RTFM`. We're using Tiny Liquid, so it is possible that wasn't implemented but I would think that would be there.

Reply:
It's short for 'read the fine manual'. 😉 
`elsif` worked. I've updated the pr. It looks a lot cleaner now. Thanks!